DDG4/edm4hep/Geant4Output2EDM4hep.cpp

```diff
@@ -0,0 +1,588 @@
+//==========================================================================
+//  AIDA Detector description implementation 
+//--------------------------------------------------------------------------
+// Copyright (C) Organisation europeenne pour la Recherche nucleaire (CERN)
+// All rights reserved.
+//
+// For the licensing terms see $DD4hepINSTALL/LICENSE.
+// For the list of contributors see $DD4hepINSTALL/doc/CREDITS.
+//
+// Author     : F.Gaede, DESY
+//
+//==========================================================================
+
+#ifndef DD4HEP_DDG4_GEANT4OUTPUT2EDM4hep_H
+#define DD4HEP_DDG4_GEANT4OUTPUT2EDM4hep_H
+
+//  Framework include files
+#include "DD4hep/Detector.h"
+#include "DD4hep/VolumeManager.h"
+#include "DDG4/Geant4HitCollection.h"
+#include "DDG4/Geant4OutputAction.h"
+#include "DDG4/Geant4SensDetAction.h"
+#include "DDG4/EventParameters.h"
+
+// Geant4 headers
+#include "G4Threading.hh"
+#include "G4AutoLock.hh"
+#include <G4Version.hh>
```

Comment:
```suggestion
#include "G4Version.hh"
```
I am a bit perplexed where do the CLHEP units come from, there is no header?

Reply:
The Units from `G4ParticleDefinition.hh` -> `PhysicalConstants.h` -> `SystemOfUnits.h` it would still good to add the header explicitly.